ui/src/app/views/profile-page/component.tsx

```diff
@@ -0,0 +1,314 @@
+              >
+                <ClrIcon
+                  shape='info-standard'
+                  className='is-solid'
+                  style={{marginLeft: 10, verticalAlign: 'middle', color: colors.blue[0]}}
+                />
+              </TooltipTrigger>
+            </React.Fragment>,
+            valueKey: 'areaOfResearch',
+            isLong: true
+          })}
+          {makeProfileInput({
+            title: 'About You',
+            valueKey: 'aboutYou',
+            isLong: true
+          })}
+          <div style={{...styles.h1, marginBottom: 24}}>Institution Affiliations</div>
+          {institutionalAffiliations.map((v, i) =>
+            <div style={{display: 'flex'}} key={`institution${i}`}>
+              {makeProfileInput({
+                title: 'Institution',
+                valueKey: `institutionalAffiliations.${i}.institution`
+              })}
+              {makeProfileInput({
+                title: 'Role',
+                valueKey: `institutionalAffiliations.${i}.role`
+              })}
+              <Clickable
+                style={{alignSelf: 'center'}}
+                onClick={() => this.setState(fp.set('profileEdits.institutionalAffiliations',
+                  fp.without([v], institutionalAffiliations)))}
+              >
+                <ClrIcon
+                  shape='times'
+                  size='24'
+                  style={{color: colors.red, marginBottom: 17}}
+                />
+              </Clickable>
+            </div>
+          )}
+          <div style={{display: 'flex', width: 520, alignItems: 'center'}}>
+            <div style={{border: `1px solid ${colors.gray[4]}`, flex: 1}}/>
+            <Clickable
+              onClick={() => this.setState(fp.set('profileEdits.institutionalAffiliations',
+                fp.concat(institutionalAffiliations, {institution: '', role: ''})))}
+            >
+              <ClrIcon
+                shape='plus-circle'
+                size='19'
+                style={{
+                  color: colors.purple[0],
+                  margin: '0 14px',
+                  flex: 'none', verticalAlign: 'text-bottom' // text-bottom makes it centered...?
+                }}
+              />
+            </Clickable>
+            <div style={{border: `1px solid ${colors.gray[4]}`, flex: 1}}/>
+          </div>
+          <div style={{marginTop: 100}}>
+            <Button
+              type='purpleSecondary'
+              onClick={() => this.setState({profileEdits: profile})}
+            >
+              Discard Changes
+            </Button>
+            <TooltipTrigger
+              side='top'
+              content={!!errors && 'You must correct errors before saving.'}
+            >
+              <Button
+                type='purplePrimary'
+                style={{marginLeft: 40}}
+                onClick={() => this.saveProfile()}
+                disabled={!!errors}
```

Comment:
Should this have a different treatment when you haven't made any changes? e.g. disabled